docs/articles/index.html

```diff
@@ -0,0 +1,140 @@
+            </span>
+            <div class="description">
+                <p>You can write TestCafe tests in ES2016 using all latest JavaScript features like 'async/await'.</p>
+                <p>TestCafe introduces simple, but powerful test API. It offers a couple of dozen methods covering all user actions. Chained syntax produces code that is easy to write and read. Furthermore, you are free to use any assertion library to perform verifications of different kinds.</p>
+                <pre><code>import { expect } from 'chai';
+
+fixture `My Fixture`
+    .page('http://devexpress.github.io/testcafe/example');
+
+test('My Test', async t => {
+await t
+    .click('#send-button')
+    .handleAlert()
+    .typeText('#input', 'Peter Parker')
+    .wait(1000);
+
+expect(await t.eval(() => getSomethingOnTheClient())).to.be.true;
+});</code></pre>
+                <p>TestCafe will compile your test code on-flight and run it immediately. It also ships with built-in support for source maps to leverage debugging experience. Source maps are automatically enabled, so all you need to do is start a debugging session in an IDE that supports them.</p>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Write Regular Client JavaScript to Observe Page State
+            </span>
+            <div class="description">
+                <p>TestCafe is capable of executing code on the client side thus giving you direct access to DOM elements on the page and powering you to obtain some data from the client.
+Simply write JavaScript code within the `Selector` or `ClientFunction` function. These functions are called from a test as regular async functions. So, you can pass parameters to it and get the function result.</p>
+
+                <pre><code>import { Selector } from 'testcafe';
+
+const getElementById = Selector(id => document.querySelector(`#${id}`));
+
+test('My Test', async t => {
+await t
+    .typeText('#developer-name', 'John Smith')
+    .click('#submit-button');
+
+const articleHeader = await getElementById('article-header');
+const headerText = articleHeader.innerText;
+});</code></pre>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Descriptive Reports
+            </span>
+            <div class="description">
+                <p>TestCafe automatically generates full-detailed reports providing a test run summary and comprehensive information about errors. Fancy call sites, clean stacks and screenshots help you easily detect an error cause.</p>
+                <p>Choose from five <a href="documentation/using-testcafe/common-concepts/reporters.html">built-in reporters</a> to output test results or create <a href="documentation/extending-testcafe/custom-reporter-plugin/">custom reporter plugins</a> to produce your own reports.</p>
+            </div>
+            <div class="gallery-container">
```

Comment:
Let's completely disable auto playback in gallery. It's painful to read text above then something is constantly scrolling below